temporalcli/commands.go

```diff
@@ -394,6 +394,10 @@ func (c *TemporalCommand) initCommand(cctx *CommandContext) {
 	}
 }
 
+func VersionString() string {
+	return fmt.Sprintf("CLI %s (Server %s, UI %s)", Version, headers.ServerVersion, version.UIVersion)
```

Comment:
Arguably we shouldn't show UI version if they haven't enabled the UI, but no big deal